### ISSUE TITLE
Port 8086 not listening with https enabled

### ISSUE THREAD

**Issue by theirman** [influxdata/influxdb#19743]
Hello

I followed this documentation to enable https in influxdb: [https://docs.influxdata.com/influxdb/v1.8/administration/https_setup/](https://docs.influxdata.com/influxdb/v1.8/administration/https_setup/). To achieve this security, I use a certificate signed by a certification authority _(sectigo)_. When I restart influxd, the service responds that it is active and running. Good!

Now, when I run my command `influx -ssl -host sicpa-iot.cati.inrae.fr -username user1 -password 'password1'`, I get the following error:
```
Failed to connect to http://sicpa-iot.cati.inrae.fr:8086: Get http://sicpa-iot.cati.inrae.fr:8086/ping: dial tcp 147.100.187.185:8086: connect: connection refused
Please check your connection settings and ensure 'influxd' is running.
```

However influxdb is running:
```
● influxdb.service - InfluxDB is an open-source, distributed, time series database
   Loaded: loaded (/lib/systemd/system/influxdb.service; enabled; vendor preset: enabled)
   Active: active (running) since Wed 2020-10-14 12:01:04 CEST; 20min ago
     Docs: https://docs.influxdata.com/influxdb/
 PID hand: 308 (influxd)
    Tasks: 13 (limit: 4915)
   Memory: 163.2M
   CGroup: /system.slice/influxdb.service
           └─308 /usr/bin/influxd -config /etc/influxdb/influxdb.conf
```

When I perform a netstat, I notice that port 8086 is not listening :
```
Activate Internet connections (only servers)
Proto Recv-Q Send-Q Local Address Foreign Address State PID/Program name
tcp 0 0 127.0.0.1:25 0.0.0:* LISTEN 10775/exim4
tcp 0 0 0.0.0.0:1883 0.0.0:* LISTEN 15217/mosquitto
tcp 0 0 127.0.0.1:6379 0.0.0:* LISTEN 15375/redis-server
tcp 0 0 0.0.0.0:22 0.0.0:* LISTEN 7466/sshd
tcp 0 0 127.0.0.1:8088 0.0.0:* LISTEN 530/influxd
tcp 0 0 127.0.0.1:5432 0.0.0.0:* LISTEN 27498/postgres
tcp6 0 0 ::1:25 :::* LISTEN 10775/exim4
tcp6 0 0 :::443 :::* LISTEN 25732/apache2
tcp6 0 0 :::1883 :::* LISTEN 15217/mosquitto
tcp6 0 0 :::8000 :::* LISTEN 21767/chirpstack-ne
tcp6 0 0 :::8001 :::* LISTEN 21783/chirpstack-ap
tcp6 0 0 :::8003 :::* LISTEN 21783/chirpstack-ap
tcp6 0 0 ::1:6379 :::* LISTEN 15375/redis-server
tcp6 0 0 :::8080 :::* LISTEN 21783/chirpstack-ap
tcp6 0 0 :::80 :::* LISTEN 25732/apache2
tcp6 0 0 :::22 :::* LISTEN 7466/sshd
tcp6 0 0 :::8888 :::* LISTEN 17837/chronograf
udp 0 0 0.0.0.0:68 0.0.0:* 407/dhclient
udp 0 0 0.0.0.0:68 0.0.0:* 379/dhclient
udp 0 0 192.0.8.13:123 0.0.0.0:* 10481/ntpd
udp 0 0 127.0.0.1:123 0.0.0.0:* 10481/ntpd
udp 0 0 0.0.0.0:123 0.0.0:* 10481/ntpd
udp6 0 0 fe80::f816:3eff:fec:123 :::* 10481/ntpd
udp6 0 0 ::1:123 :::* 10481/ntpd
udp6 0 0 :::123 :::* 10481/ntpd
udp6 0 0 :::1700 :::* 21747/chirpstack-ga
```

Here is the [http] section of my influxdb.conf configuration file:
```
[http]
  # Determines whether HTTP endpoint is enabled.
  enabled = false

  # The bind address used by the HTTP service.
  bind-address = ":8086"

  # Determines whether user authentication is enabled over HTTP/HTTPS.
  auth-enabled = true

  # Determines whether HTTPS is enabled.
  https-enabled = true

  # The SSL certificate to use when HTTPS is enabled.
  https-certificate = "/etc/ssl/sicpa-iot_cati_inrae_en/sicpa-iot_cati_inrae_fr_bundle.crt"

  # Use a separate private key location.
  https-private-key = "/etc/ssl/sicpa-iot_cati_inrae_en/sicpa-iot_cati_inrae_en.key".
```

What do you think is wrong?

Sincerely
Thierry

**Comment by psteinbachs** [comment]
`enabled = false` needs to be set to `enabled = true` for either http or https to be listening.  If set to true and `https-enabled = true` is also set, then only https will be active.  Our docs need to be updated to point out this nuance.

**Comment by theirman** [comment]
Thank you for your answer @psteinbachs

Indeed, it's not very clear. I thought that `https-enabled = true` activated https and, therefore, that we had http/https active. After that, I said to myself that I didn't want the possibility of seeing data going back up in http so I changed it to `enabled = false`.

I just reactivated `enabled = true` and it works again. Even knowing what you've just explained, reading the configuration remains disturbing for a beginner like me

Many thanks again !
